### ISSUE TITLE
Python version needed to build

### ISSUE THREAD

**Issue by IrishBumfuzzle** [jellyfin/jellyfin-mpv-shim#263]
**Is your feature request related to a problem? Please describe.**
In the readme.md, the version of python required to build the project is not mentioned. As currently the pythonnet dependency is only available till v3.8

**Describe the solution you'd like**
Mention the python version would be great.

**Describe alternatives you've considered**

**Additional context**

**Comment by IrishBumfuzzle** [comment]
Also this line need to be edited, `Extract the mpv-1.dll from the file and move it to the jellyfin-mpv-shim folder.` in building for windows to `mpv-2.dll`

**Comment by iwalton3** [comment]
You can use older versions of those packages. I had to upgrade it due to issues with CI builds. You can also just omit some optional dependencies but it’ll run with reduced functionality.

Last I recall this will work with a version as old as Python 3.6.

**Comment by IrishBumfuzzle** [comment]
> You can use older versions of those packages. I had to upgrade it due to issues with CI builds. You can also just omit some optional dependencies but it’ll run with reduced functionality.
> 
> Last I recall this will work with a version as old as Python 3.6.

I mean to say that it doesn't build with new versions of python like 3.10.

**Comment by iwalton3** [comment]
Oh if you are building for windows it is kind of a mess too. Your best bet until I get a new release actually finished is read the CI code.

**Comment by IrishBumfuzzle** [comment]
I'll make do, thanks.

**Comment by iwalton3** [comment]
Yeah unfortunately the windows build seems to break every time I go to make changes to this project as of late.
